Gems/LyShine/Code/Source/Animation/AnimTrack.h

```diff
@@ -221,6 +221,8 @@ class TUiAnimTrack
     float m_lastTime;
     int m_flags;
 
+    constexpr unsigned int InvalidKey = 0x7FFFFFFF;
```

Comment:
uber nit: Might not be worth it but this could be `uint32_t` instead of `unsigned int` (or `AZ::u32` but I think the standard one is now preferred last I heard)

Reply:
This is being assigned to `m_currKey`:
```
unsigned int m_currKey : 31;
```
So I prefer to match the exact type to avoid data loss assignments (it shouldnt since in all platforms `unsigned int` == `uint32_t`, but better to keep the same types). Changing `m_currKey` type would go out of the scope of this change... (we are using `int` and `unsigned int` all over the place in LyShine, Maestro and Editor

Reply:
Okay understood, totally makes sense. Thanks!